src/libs/ReportUtils.js

```diff
@@ -394,7 +394,7 @@ function isUserCreatedPolicyRoom(report) {
  * @returns {Boolean}
  */
 function isPolicyExpenseChat(report) {
-    return getChatType(report) === CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT;
+    return getChatType(report) === CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT || lodashGet(report, 'isPolicyExpenseChat', false);
```

Comment:
The most recent guidelines favor "modern" syntax (`?.`) over `lodashGet`, but it might work only for `.ts` files. I'm not 100% sure

Reply:
👍

Reply:
@cubuspl42, after the latest sync with the `main`, I've applied the new approach.